.gitlab/e2e/e2e.yml

```diff
@@ -322,3 +322,12 @@ new-e2e-ndm-netflow:
   variables:
     TARGETS: ./tests/ndm/netflow
     TEAM: network-device-monitoring
+
+new-e2e-ndm-snmp:
+  extends: .new_e2e_template
+  rules: !reference [.on_ndm_snmp_or_e2e_changes_or_manual]
+  needs:
+    - qa_agent
+  variables:
+    TARGETS: ./tests/ndm/snmp
```

Comment:
When running the new-e2e ndm/snmp test in CI I believe it can/will hit the DockerHub rate limit pulling datadog/docker-library:snmp.   We should probably create our own docker workload app for that container in the Datadog Github Registry like we did for the new-e2e ndm/netflow test (https://github.com/DataDog/test-infra-definitions/tree/main/components/datadog/apps/netflow-generator/images/netflow-generator).

Reply:
created with [this pr](https://github.com/DataDog/test-infra-definitions/pull/849)

Reply:
Following [the comment of kevin](https://github.com/DataDog/test-infra-definitions/pull/849#issuecomment-2136991671) we are going to use the container stored in cache instead (https://github.com/DataDog/datadog-agent/pull/26003/commits/114b9cb88ebd9c10a54bdd1bd821952036daf794)
I will do another pr to do the same for netflow to be consistent

Reply:
Even better - thx!